apitools/org.eclipse.pde.api.tools/src/org/eclipse/pde/api/tools/internal/model/ArchiveApiTypeContainer.java

```diff
@@ -238,7 +238,7 @@ public String[] getPackageNames() throws CoreException {
 		init();
 		synchronized (this) {
 			if (fPackageNames == null) {
-				fPackageNames = fPackages.keySet().toArray(String[]::new);
```

Comment:
@merks in b973ca8f012ff6de044b90485f38685bc3fef8b6 you changed this from sorted to not sorted. Was this intentional?

Reply:
It's initialized by `fPackages = new TreeMap<>();`.

Reply:
May bad. Thanks for the clarification! Checked the other implementations as well, but that's the only one where sorting is not necessary.